printer_mods/README.md

```diff
@@ -69,6 +69,7 @@ like so:
 | Conqueror433 | [Voron Parts Logo](./conqueror433/Voron_Parts_Logo) | Voron Parts Logo for your back panels, 3D Print or Laser Cut | :x: | :x: | :heavy_check_mark: | :x: |
 |consibonsi | [Y-Mod SW 9mm](./consibonsi/Y-Mod_SW_9mm)| 9mm Y belt mod for SW | :x: |:x: |:x: |:heavy_check_mark:|
 | d3nny | [Filtered Power Plug with RND_165-00057](./d3nny/mains_plug_panel_for_RND_165-00057) | Filtered Power Plug for RND 165-00057 and CAT6-Plug | :x: | :x: | :heavy_check_mark: |:x:|
+|DaGlo| [V0.1 Duet 2 Wifi mounting brackets](./DaGlo/Duet2Wifi_Mounting_Bracket/) | Voron 0.1 Mounting brackets with/without RPi 4 | :heavy_check_mark: | :x: | :x: | :x: |
```

Comment:
Broken link, your directory is called `Voron_V0_Duet_2_WiFi_Mounting_Brackets`, but the link goes to `Duet2Wifi_Mounting_Bracket`.